src/main/java/org/datavyu/plugins/ffmpeg/NativeMediaPlayer.java

```diff
@@ -627,6 +640,40 @@ final public void dispose() {
         }
     }
 
+    @Override
+    public synchronized void updateMasterTime(final double masterClockTime) {
+        if (!isDisposed) {
+            masterCurrentTime = masterClockTime / 1000;
+            double presentationTime = playerGetPresentationTime();
+
+            if (presentationTime >= 0.0
+                    && (Double.compare(presentationTime, playerPreviousTime) != 0
```

Comment:
Sorry, this should be masterCurrentTime instead of playerPreviousTime. 

The condition is meant to enable fake playback by comparing the presentation time of the stream against the master clock, where the stream is stopped and the master clock is running, in this case, the condition Double.compare(presentationTime, masterCurrentTime ) != 0 will be true and we force a sync